docs/docs/HowTo/schemas/Request-a-schema.md

```diff
@@ -0,0 +1,59 @@
+# Request a new schema
+
+You can request a new schema to be created in Kafka quickly and easily
+using Klaw. If you own a topic, you will automatically become the schema
+owner. As the topic owner, you have the ability to create, approve, or
+decline any schema request.
+
+## Prerequisites
+
+-   The Superadmin must configure the environment within Klaw. See
+    [Environments](../../Concepts/clusters-environments) for the steps to configure the environments.
+
+## Make a schema request
+
+1.  Navigate to the **Topics** menu in Klaw and select the topic for
+    which you want to request a schema.
+2.  Select the **Schemas** tab, and **Request Schema** button in the
+    bottom right corner.
+3.  Select the schema environment in which you want the schema to be
+    created.
+
+>In most cases, you must start with the lowest environment and use the
+promotion feature for higher environments.
+
+4.  If not already selected, choose the topic on which the schema will
+    be registered. Keep in mind that, as per the `TopicNameStrategy`,
+    the schema is created on the subject-value, where the subject is the
+    topic name.
+
+>Registering schemas on subject-key is not possible in Klaw. Klaw
```

Comment:
Similar situation to promote a schema
> not rendering and also it looks like bullet points arent rendering correctly.

![image](https://github.com/aiven/klaw-docs/assets/121855584/44df7574-d24b-429a-abb3-3d586cb39270)


here is what old documentation looks like. (bullet points look right but the '>' seem to be notes.)

![image](https://github.com/aiven/klaw-docs/assets/121855584/fdd60c59-56cd-486a-aa0b-d1f8978d8e83)